.github/workflows/zero-downtime-tests.yml

```diff
@@ -9,6 +12,7 @@ on:
       upgradeTo:
         description: 'The helm upgrade command will run with this version (e.g. 0.15.0-dev-PR-7504)'
         required: true
+        default: 'https://github.com/keptn/helm-charts-dev/raw/69eea439a26a99ecc163e296860dbb5d43e41600/packages/keptn-0.15.1-dev.tgz'
```

Comment:
i dont think this will work with `helm upgrade keptn --version upgradeTo`.. you will need to remove the `--version` i think